tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -86,6 +91,7 @@ class RetryingWritableFile : public WritableFile {
 
  private:
   std::unique_ptr<WritableFile> base_file_;
+  int initial_delay_seconds;
```

Comment:
Please add a trailing underscore to the name, see this section for details
https://google.github.io/styleguide/cppguide.html#Variable_Names

Please also add a 'const' modifier, as this member is not supposed to be changed in the lifetime of the object.

Also applies to other places where you're adding the new member.